rhel8/profiles/ospp.profile

```diff
@@ -8,848 +8,534 @@ description: |-
     Operating Systems (Protection Profile Version 4.2).
 
 selections:
+    #################################################################
+    ## BOOT
+    #################################################################
 
-    #
-    # Set the system's root password (required)
-    # Plaintext password is: server
-    # Refer to https://pykickstart.readthedocs.io/en/latest/commands.html#rootpw
-    # to see how to create an encrypted password for a different plaintext password
-    #rootpw --iscrypted $6$rhel6usgcb$aS6oPGXcPKp3OtFArSrhRwu6sN8q2.yEGY7AIwDOQd23YCtiz9c5mXbid1BzX9bmXTEZi.hCzTEXFosVBI5ng0
-
-    # This profile will restrict root login. Add a user that can login and
-    # change to root.
-    # Plaintext password is: admin123
-    #user --name=admin --groups=wheel --password=$6$Ga6ZnIlytrWpuCzO$q0LqT1USHpahzUafQM9jyHCY9BiE5/ahXLNWUMiVQnFGblu0WWGZ1e6icTaCGO4GNgZNtspp1Let/qpM7FMVB0 --iscrypted
+    ## RHEL 8 CCE-80829-5: Set the UEFI Boot Loader Password
+    - grub2_uefi_password
 
 
     #################################################################
-    ## SELinux Configuration
+    ## Systemd Items
     #################################################################
 
-    ## RHEL 8 CCE-80869-1: Ensure SELinux is Enforcing
-    - var_selinux_state=enforcing
-    - selinux_state
+    ## RHEL 8 CCE-80855-0: Require Authentication for Single User Mode
+    - require_singleuser_auth
 
-    ## RHEL 8 CCE-80868-3: Configure SELinux Policy
-    - var_selinux_policy_name=targeted
-    - selinux_policytype
```

Comment:
NACK. Looks like several rules were removed, such as SELinux. Need to refactor and add them all back in.

Reply:
but SELinux rules shouldn't be there

Reply:
and SSHD ciphers and MACs should be solved by crypto-policies in future

Reply:
It's a required part of our OSPP profile.

Reply:
@shawndwells Can you please provide any references? ... SELinux is not part of NIAP's OSPPv4.2.1 nor is it implied in our ST. Not having it in the profile doesn't mean it's disabled, just that it's not checked/remediated.

Reply:
OSPP 4.2 can be found here:
https://www.niap-ccevs.org/MMO/PP/-442-/#SecurityObjectivesTOE

With the config annex:
https://www.niap-ccevs.org/MMO/PP/-442ConfigAnnex-/

NIAP made it clear they will reject any certification package lacking SELinux. And I've made it clear I will reject any engineering work lacking SELinux as well.

Reply:
@jan-cerny Can you add the selinux rules back? So we can get this merged?

Reply:
@yuumasato yes, I'm adding them back right now.

Reply:
@yuumasato @shawndwells I have re-added  SELinux rules in 98160f6